.github/CONTRIBUTING.md

```diff
@@ -24,10 +24,12 @@ To confirm your pull request (PR) passes the Travis pylint test, the following
 test must run without any output:
 
 ```bash
-$ workon YOUR_VIRTUALENV
+$   workon YOUR_VIRTUALENV
 
 # Optional
```

Comment:
We should not do this and instead redirect them to having our container / unit tests ran.